packages/expo/cli/utils/exit.ts

```diff
@@ -1,7 +1,12 @@
-/** Add functions that run before the process exits. */
-export function installExitHooks(listener: NodeJS.SignalsListener) {
+/** Add functions that run before the process exits. Returns a function for removing the listeners. */
+export function installExitHooks(listener: NodeJS.SignalsListener): () => void {
   const killSignals: ['SIGINT', 'SIGTERM'] = ['SIGINT', 'SIGTERM'];
   for (const signal of killSignals) {
     process.on(signal, listener);
   }
+  return () => {
+    for (const signal of killSignals) {
+      process.off(signal, listener);
+    }
```

Comment:
Useful to let the listener be freed if the caller holds onto the function to remove the listeners even after removing them (matters more in website programming though)

```suggestion
    if (!listener) {
      return;
    }
    for (const signal of killSignals) {
      process.off(signal, listener);
    }
    // Allow the listener to be GC'd
    listener = null;
```